src/board/properties/railroadProperty.js

```diff
@@ -0,0 +1,20 @@
+var Property = require('./property'), 
+    inherits = require('util').inherits;
+
+
+// rent should be array with following format: 
+// [1 owned (rent), 2 owned, 3 owned, 4 owned]
```

Comment:
how does rent work for this? It seems that we probably don't need to pass it in because for all RRs it's 50, 100, 150, 200 right?